templates/main.template.yaml

```diff
@@ -1,1059 +1,1059 @@
-      TemplateURL: !Sub
-        - >-
-          https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/uipath-orchestrator.template.yaml
-        - S3Bucket: !If
-            - UsingDefaultBucket
-            - !Sub '${QSS3BucketName}-${AWS::Region}'
-            - !Ref QSS3BucketName
-          S3Region: !If
-            - UsingDefaultBucket
-            - !Ref 'AWS::Region'
-            - !Ref QSS3BucketRegion
-      Parameters:
-        VPCCIDR: !Ref VPCCIDR
-        VPCID: !GetAtt NetworkStack.Outputs.VPCID
-        KeyPairName: !Ref KeyPairName
-        PrivateSubnetIDs: !Join
-          - ','
-          - - !GetAtt NetworkStack.Outputs.PrivateSubnet1AID
-            - !GetAtt NetworkStack.Outputs.PrivateSubnet2AID
-            - !If [ 3AZCondition, !GetAtt NetworkStack.Outputs.PrivateSubnet3AID, !Ref "AWS::NoValue" ]
-        PublicSubnetIDs: !Join
-          - ','
-          - - !GetAtt NetworkStack.Outputs.PublicSubnet1ID
-            - !GetAtt NetworkStack.Outputs.PublicSubnet2ID
-            - !If [ 3AZCondition, !GetAtt NetworkStack.Outputs.PublicSubnet3ID, !Ref "AWS::NoValue" ]
-        InstanceType: !Ref InstanceType
-        GatewayName: !Ref GatewayName
-        GatewayTimezone: !Ref GatewayTimezone
-        UiPathFQDN: !Ref UiPathFQDN
-        HostedZoneID: !Ref HostedZoneID
-        HAAUser: !Ref HAAUser
-        HAAPassword: !Ref HAAPassword
-        HAPort: !Ref HAPort
-        HAALicense: !Ref HAALicense
-        RDSDBName: !Ref RDSDBName
-        RDSAllocatedStorage: !Ref RDSAllocatedStorage
-        RDSMaxAllocatedStorage: !Ref RDSMaxAllocatedStorage
-        RDSUsername: !Ref RDSUsername
-        RDSPassword: !Ref RDSPassword
-        OrchestratorVersion: !Ref OrchestratorVersion
-        OrchestratorInstanceCount: !Ref OrchestratorInstanceCount
-        OrchestratorLicense: !Ref OrchestratorLicense
-        OrchestratorPassword: !Ref OrchestratorPassword
-        QSS3BucketName: !Ref QSS3BucketName
-        QSS3KeyPrefix: !Ref QSS3KeyPrefix
-        QSS3BucketRegion: !Ref QSS3BucketRegion
-        BastionIngressCIDR: !Ref BastionIngressCIDR
-Outputs:
-  FQDNLoadBalancer:
-    Description: UiPath Orchestrator login page.
-    Value: !Ref UiPathFQDN
-  BastionHost:
-    Description: Bastion host public IP address.
-    Value: !GetAtt
-      - OrchestratorStack
-      - Outputs.BastionHost
-  HaaNodesIPs:
-    Description: High Availability Add-on IP addresses.
-    Value: !GetAtt
-      - OrchestratorStack
-      - Outputs.HaaNodesIPs
-  NugetStorageS3Bucket:
-    Description: S3 bucket with UiPath Orchestrator NuGet packages.
-    Value: !GetAtt
-      - OrchestratorStack
-      - Outputs.NugetStorageS3Bucket
-  Postdeployment:
-    Description: See the deployment guide for post-deployment steps.
-    Value: https://aws.amazon.com/quickstart/?quickstart-all.sort-by=item.additionalFields.sortDate&quickstart-all.sort-order=desc&awsm.page-quickstart-all=5
+AWSTemplateFormatVersion: 2010-09-09

```

Comment:
I am unable to see what has changed. It shows the entire file is changed which makes it hard for me to review the changes, without going through line by line and do a full review, which will take much longer time.
This is same for both the template files.
Is there anything you are doing that's making github not to highlight just the changed part?

Reply:
Hi Shivansh, 
Yes, I don't understand why github is showing the file like this (maybe some CRLF/LF issues). Still, I can see the changes as shown here:
![image](https://user-images.githubusercontent.com/10058111/131637495-31436cbb-39b3-41e6-819a-80fc996e10b9.png)